WordPress/Classes/Services/CommentServiceRemoteFactory.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
+import WordPressKit
+
+
+/// Provides service remote instances for CommentService
+@objc class CommentServiceRemoteFactory: NSObject {
+
+    /// Returns a CommentServiceRemote for a given Blog object
+    ///
+    /// - Parameter blog: A valid Blog object
+    /// - Returns: A CommentServiceRemote instance
+    @objc func remote(blog: Blog) -> CommentServiceRemote? {
+        if blog.supports(.wpComRESTAPI),
+            let api = blog.wordPressComRestApi(),
+            let dotComID = blog.dotComID {
+            return CommentServiceRemoteREST(wordPressComRestApi: api, siteID: dotComID)
+        } else if let api = blog.xmlrpcApi,
```

Comment:
This `else` isn't really necessary since the the `if` returns.

Reply:
Noted, will do!